kaspresso/src/main/kotlin/com/kaspersky/kaspresso/kaspresso/Kaspresso.kt

```diff
@@ -319,6 +320,24 @@ data class Kaspresso(
             }
         }
 
+        /**
+         * In some cases, scrolling to the view works, but the action cannot be performed due to padding in
+         * the scrollable view, namely
+         * - ScrollView or NestedScrollView
+         * - ListView
+         * - HorizontalScrollView
+         *
+         * This is a very special case. If any of the screens in your tests contains one of the aforementioned
+         * Views and they have padding, use this in your [Kaspresso.Builder] to avoid scrolling problems
+         */
+        fun withAutoScrollFallback(): Builder =
```

Comment:
Take time to think about how to make it better

Reply:
I've been thinking about how to solve it... honestly thinking, I strongly believe we can replace the current AutoScroll with this one, since the only difference is that we scroll a little bit more to consider the paddings.
What is your take on this? @matzuk @eakurnikov ?

Reply:
Done in the last commit!
I am pretty happy with the result!